app/views/exchange_rates/index_average_rates.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for :top_breadcrumbs do %>
```

Comment:
So I guess this is a tactical win so we can show progress to the client?

I'm kind of of the opinion we should just make this work for all future average rates in one go so we don't have to revisit this later?

Reply:
we got something fully working.
let's sheep it now, and release some value, we can refactor it soon, even tomorrow 😄